### DISCUSSION TITLE
CAISO imports

### DISCUSSION THREAD

**Discussion by intermittentnrg** [electricitymaps/electricitymaps-contrib#6785] [Q&A] [answered]
I see CAISO has already been discussed in existing issues so not sure I can add more to the discussion.

But only Mexico is showing up for CAISO imports/exports. Are they being counted correctly?

![bild](https://github.com/electricitymaps/electricitymaps-contrib/assets/96312338/99445bc5-9e11-4db3-b15e-17e0a4c3f21e)

**Comment by VIKTORVAV99** [comment, marked as answer]
Hi!

Imports and exports are being accounted for it's just that they are delayed in the source data by more time than we show in the app. I believe we get all the exchange data from EIA except for the Mexico exchange which is why that is still shown.

Here you can see a representation of the data availability from our internal tools:
<img width="715" alt="image" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/30777521/8bee7c5d-c0da-40b3-b4e7-164bb6188c74">


PS: I'll also turn this into a discussion as I believe it's more suitable to the subject.
